tests/unit/helpers/is-empty-test.js

```diff
@@ -0,0 +1,75 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import { registerHelper } from 'ember-truth-helpers/utils/register-helper';
+import {
+  isEmpty as isEmptyHelper
+} from 'ember-truth-helpers/helpers/is-empty';
+
+moduleForComponent('is-empty', 'helper:is-empty', {
+  integration: true,
+  beforeEach: function() {
+    // Do not register helpers from Ember 1.13 onwards, starting from 1.13 they
+    // will be auto-discovered.
+    if (!Ember.Helper) {
+      registerHelper('is-empty', isEmptyHelper);
+    }
+  }
+});
+
+test('it works with array operations', function(assert) {
+  let contextChild = Ember.Object.create({
+    someArray: null
+  });
+  this.set('contextChild', contextChild);
+
+  this.render(
+    Ember.HTMLBars.compile("{{is-empty contextChild.someArray}}")
```

Comment:
Suggestion!

Do you would can change `Ember.HTMLBars.compile` to `htmlbars-inline-precompile`?